frontend/src/components/OmakaseModal.jsx

```diff
@@ -0,0 +1,209 @@
+import { useState } from "react";
+import {
+  Modal,
+  Spinner,
+  FormField,
+  Select,
+  Input,
+  Checkbox,
+  ExpandableSection,
+  SpaceBetween,
+  Button,
+  Box,
+} from "@cloudscape-design/components";
+import useStore from "../stores/useStore";
+import useOmakaseStore from "../stores/useOmakaseStore";
+import { createInitialFormData, executeExport } from "../utils/omakase";
+
+export default function OmakaseModal({ editTimeranges, flows }) {
```

Comment:
nit: i wonder about renaming this OmakaseExportModal?

Reply:
good idea 👍